### PR TITLE
Fix remove-comments

### PR DESCRIPTION
**Pull request by ThistleSifter**
I had a few spare minutes this morning so I thought I'd have a go at a couple of bundler issues.

First is the issue with multiline comments that contain a `]`.

I also added a test. I'm not sure if that's the kind of test you're after or if you want more. Let me know.

### CODE REVIEW

**Review by Nick-Mazuk — APPROVED** [review]
Looks good to me. Since I just approved and merged another bundler PR, this one will need to be rebased and rebuilt (with `pnpm build`).

Once you do that, feel free to merge.

**Comment by ThistleSifter** [comment]
I have rebased, re-added the call to `removeComments` to the bundler and rebuilt.
(I don't have merge permissions)

**Comment by Nick-Mazuk** [comment]
Merging, let's hope this doesn't break things like my other attempts at this.

**Pull request merged**
